### ISSUE TITLE
Trouble following packaging libraries tutorial

### ISSUE THREAD

**Issue by IOnlyPlayAsDrift** [pypa/packaging-problems#349]
1. Windows 10

2. Python 3.7.4

3. pip 20.1

4. https://packaging.python.org/tutorials/packaging-projects/

5. I'm trying to upload my Snookey2 module to pypi but for some reason I get the error "error: [Errno 13] Permission denied: 'Snookey2'" and I don't think it's permission problems because I got on a administrator account and it still gives me this error. Here's the whole log of it:
C:\Users\djane\OneDrive\Desktop\snookey2>python setup.py sdist bdist_wheel
running sdist
running egg_info
creating Snookey2.egg-info
writing Snookey2.egg-info\PKG-INFO
writing dependency_links to Snookey2.egg-info\dependency_links.txt
writing requirements to Snookey2.egg-info\requires.txt
writing top-level names to Snookey2.egg-info\top_level.txt
writing manifest file 'Snookey2.egg-info\SOURCES.txt'
reading manifest file 'Snookey2.egg-info\SOURCES.txt'
writing manifest file 'Snookey2.egg-info\SOURCES.txt'
running check
warning: Check: missing meta-data: if 'author' supplied, 'author_email' must be supplied too

creating Snookey2-4.0
creating Snookey2-4.0\Snookey2.egg-info
copying files to Snookey2-4.0...
copying README.md -> Snookey2-4.0
'Snookey2' not a regular file -- skipping
copying setup.py -> Snookey2-4.0
copying Snookey2.egg-info\PKG-INFO -> Snookey2-4.0\Snookey2.egg-info
copying Snookey2.egg-info\SOURCES.txt -> Snookey2-4.0\Snookey2.egg-info
copying Snookey2.egg-info\dependency_links.txt -> Snookey2-4.0\Snookey2.egg-info
copying Snookey2.egg-info\requires.txt -> Snookey2-4.0\Snookey2.egg-info
copying Snookey2.egg-info\top_level.txt -> Snookey2-4.0\Snookey2.egg-info
Writing Snookey2-4.0\setup.cfg
Creating tar archive
removing 'Snookey2-4.0' (and everything under it)
running bdist_wheel
running build
running build_scripts
error: [Errno 13] Permission denied: 'Snookey2'

and then the setup.py file says this:
from setuptools import setup

with open("README.md", "r") as fh:
    long_description = fh.read()

setup(name='Snookey2',
      author='IOnlyPlayAsDrift',
      version='4.0',
      description='Stream on RPAN on PC easily using OBS and this module!',
      long_description=long_description,
      long_description_content_type="text/markdown",
      url='https://github.com/IOnlyPlayAsDrift/Snookey2',
      install_requires=[
          'pytz',
          'requests', 
          'pypresence'
      ],
      scripts=['Snookey2']
      ) 

Let me know if there's any more info I need to supply and what I need to do to fix this. Thanks!

**Comment by uranusjr** [comment]
It’s discouraged to use the `scripts` declarative. Consider rewriting it with [`entry_points`](https://setuptools.readthedocs.io/en/latest/setuptools.html#automatic-script-creation) instead.

**Comment by merwok** [comment]
The error message seem to indicate that there is no file named `Snookey2` so your `scripts` entry cannot be processed.  Can you check that this file exists and is readable?

**Comment by IOnlyPlayAsDrift** [comment]
> The error message seem to indicate that there is no file named `Snookey2` so your `scripts` entry cannot be processed. Can you check that this file exists and is readable?

Yes the Snookey2 folder exists, I’ve tried removing it and making a new folder and nothing works.

> It’s discouraged to use the `scripts` declarative. Consider rewriting it with [`entry_points`](https://setuptools.readthedocs.io/en/latest/setuptools.html#automatic-script-creation) instead.

and thanks I’ll try that!

**Comment by pfmoore** [comment]
> Yes the Snookey2 folder exists

But using it in the "scripts" entry requires that it's a file, so that's why you're getting the permission error (setuptools is trying to open the directory as a file).
